src/runtimes/webworker.ts

```diff
@@ -57,6 +60,12 @@ export const webworker = (props: RuntimeProps) => {
         },
     }
 
+    const chat: Chat = {
```

Comment:
This implementation of Chat interface is not complete. What's the plan for finishing it?

Reply:
Currently I would call it as a minimum viable product, it is sufficient for creating a simple chat server and we can continue iterating on it